recipes/jupysql/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "jupysql" %}
+{% set version = "0.6.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupysql-{{ version }}.tar.gz
+  sha256: 8c328e18ff43f566905aaa8a26c7c4693d83be5ba5e69bd2bb6c1d6a3c3af865
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - prettytable
+    - ipython >=1.0
+    - sqlalchemy <2
+    - sqlparse
+    - ipython_genutils >=0.1.0
+    - jinja2
+    - ploomber-core >=0.2.4
+    - importlib-metadata  # [py<38]
```

For noarch (better to just unconditionally depend on `importlib-metadata`):
```suggestion
    - python >=3.7
    - pip
  run:
    - python >=3.7
    - prettytable
    - ipython >=1.0
    - sqlalchemy <2
    - sqlparse
    - ipython_genutils >=0.1.0
    - jinja2
    - ploomber-core >=0.2.4
    - importlib-metadata
```